### PR TITLE
LPS-148909 Wrong message showing on Form Builder after enabling the see partial results settings

### PR DESCRIPTION
**Pull request by daiazius**
Related issue: https://issues.liferay.com/browse/LPS-148909

### CODE REVIEW

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by alinedoleron** [comment]
Hi @daiazius. I've tested in my machine and the messages are being displayed as keys (for example: respondents-can-see-all-submitted-form-data) and not the message. Did this code worked for you?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/util/ShowPartialResultsAlert.tsx [hunk 1]
```diff
@@ -27,6 +27,10 @@ const ShowPartialResultsAlert: React.FC<IProps> = ({
 		!showPartialResultsToRespondents
 	);
 
+	const showPartialResultsMessage = dismissible
+		? 'your-responses-will-be-visible-to-all-form-respondents'
+		: 'respondents-can-see-all-submitted-form-data';
+
```
Comment (alinedoleron) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/util/ShowPartialResultsAlert.tsx:33]: Hi @daiazius, as I talked to you on Slack, Liferay.Language.get function doesn't work well with variables. Please, could you test is this way it works?
```suggestion
		const showPartialResultsMessage = dismissible
	? Liferay.Language.get('your-responses-will-be-visible-to-all-form-respondents')
	: Liferay.Language.get('respondents-can-see-all-submitted-form-data');

```

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/util/ShowPartialResultsAlert.tsx [hunk 2]
```diff
@@ -38,9 +42,7 @@ const ShowPartialResultsAlert: React.FC<IProps> = ({
 				onClose={dismissible ? () => setDismissed(true) : undefined}
 				title="Info"
 			>
-				{Liferay.Language.get(
-					'your-responses-will-be-visible-to-all-form-respondents'
-				)}
+				{Liferay.Language.get(showPartialResultsMessage)}
```
Comment (alinedoleron) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/util/ShowPartialResultsAlert.tsx:45]: Also change this line to:

```suggestion
				{showPartialResultsMessage}
```

**Review by alinedoleron — CHANGES_REQUESTED** [review]

**Review by alinedoleron — APPROVED** [review]

**Comment by daiazius** [comment]
ci:forward

**Comment by alinedoleron** [comment]
ci:forward

**Comment by carolmariaabb** [comment]
ci:forward

**Comment by daiazius** [comment]
ci:forward

**Comment by HenriquePereiraa** [comment]
ci:forward

**Comment by alinedoleron** [comment]
ci:forward

**Comment by alinedoleron** [comment]
Hi @HenriquePereiraa. Please, could you run `yarn test` on `dynamic-data-mapping-form-web` module?

**Comment by alinedoleron** [comment]
This is the new [PR](https://github.com/liferay-forms/liferay-portal/pull/2018) fixing the tests.

**Pull request closed without merging**